src/slider/slider.component.ts

```diff
@@ -242,6 +248,7 @@ export class Slider implements AfterViewInit, OnDestroy, ControlValueAccessor {
 			this.input.classList.add("bx--slider-text-input");
 			this.input.classList.add("bx--text-input");
 			this.input.setAttribute("aria-labelledby", `${this.bottomRangeId} ${this.topRangeId}`);
+			this.input.setAttribute("aria-label", this.ariaLabel);
```

Comment:
I don't think this is needed since we already set `labelledby`

Reply:
Plus the input is passed in from the outside - the consumer should manage most of the aria attributes on that side